update colormap docs (#383)
cogeotiff/rio-tiler: docs/colormap.md

Rio-tiler includes many colormaps, some derived from Matplotlib and some custom
ones that are commonly used with raster data.

> You can load one of rio-tiler's default colormap using `rio_tiler.colormap.cmap` object, and then pass it

Reviewer: ```suggestion
You can load one of `rio-tiler`'s default colormaps from the `rio_tiler.colormap.cmap` object, and then pass it
```

to `rio_tiler.utils.render`:

```python
from rio_tiler.colormap import cmap
from rio_tiler.io import COGReader

# Get Colormap

> cm = cmap.get("cfastie")

Reviewer: ```suggestion
# You can list available colormap names with `cmap.list()`
cm = cmap.get("cfastie")
```
